shared/Microsoft.Extensions.CopyOnWriteDictionary.Sources/CopyOnWriteDictionary.cs

```diff
@@ -13,6 +13,10 @@ internal class CopyOnWriteDictionary<TKey, TValue> : IDictionary<TKey, TValue>
         private readonly IEqualityComparer<TKey> _comparer;
         private IDictionary<TKey, TValue> _innerDictionary;
 
+        public CopyOnWriteDictionary(IDictionary<TKey, TValue> sourceDictionary)
```

Comment:
That's not a common scenario. We usually are working with IDictionary. I think there's nothing that needs to be added here. Will close.